### PR TITLE
finish videos routes

### PR DESCRIPTION
**Pull request by kevinleaves**
- implement RESTFUL routes for /videos

### CODE REVIEW

File: src/server/models/videos.js [hunk 1]
```diff
@@ -1 +1,45 @@
...
+  getAllVideos: async () => {
+    try {
+      const db = await dbPromise;
+      const videoRef = collection(db, 'videos');
+      const querySnapshot = await getDocs(videoRef);
+      const documents = querySnapshot.docs.map((doc) => ({ id: doc.id, ...doc.data() }));
+      return documents;
+    } catch (err) {
+      console.error(err);
+    }
```
Comment (BrettEastman) [inline src/server/models/videos.js:15]: Needs return err after console.error

File: src/server/models/videos.js [hunk 2]
```diff
@@ -1 +1,45 @@
...
+
+const dbPromise = require('../../config/firebase');
+
+module.exports = {
+  getAllVideos: async () => {
+    try {
+      const db = await dbPromise;
+      const videoRef = collection(db, 'videos');
+      const querySnapshot = await getDocs(videoRef);
+      const documents = querySnapshot.docs.map((doc) => ({ id: doc.id, ...doc.data() }));
```
Comment (BrettEastman) [inline src/server/models/videos.js:11]: Getting a few linter errors. I believe the doc parameter should be named something else

**Review by BrettEastman — CHANGES_REQUESTED** [review]
Looks good in general, but I'm getting a few linter errors. Fix those and we should be good.

File: src/server/models/videos.js (hunk 1 above)
Comment (davidzny195) [reply src/server/models/videos.js:15]: I'm gonna merge it and fix the errors, because it seems kevin has the updated db.js code

**Review by davidzny195 — APPROVED** [review]

**Pull request merged**